app/api/routers/automatic.py

```
@@ -0,0 +1,127 @@
+
+        room_pool = get_room_pool()
+        room_stats = await room_pool.get_pool_stats()
+
+        return JSONResponse(
+            {
+                "status": "healthy",
+                "voice_pool_stats": voice_stats,
+                "room_pool_stats": room_stats,
+            }
+        )
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)